packages/core/src/Avatar/Avatar.styles.tsx

```diff
@@ -25,7 +25,7 @@ export const { staticClasses, useClasses } = createClasses("HvAvatar", {
     display: "flex",
     alignItems: "center",
     position: "relative",
-    padding: theme.space.xxs,
+    padding: 3,
```

Comment:
where is this `3` coming from? we should avoid using values outside our spacing tokens
there's some miss-alignment now:
![image](https://github.com/user-attachments/assets/7a5b3148-631f-44bb-aca0-6f2c90f9c8a1)

Reply:
the spec says `4px`
![image](https://github.com/user-attachments/assets/a8db3b65-83b1-4148-bc3a-aec37590455b) ![image](https://github.com/user-attachments/assets/0b7b2d47-47f0-4cd5-9124-c1c8202d94a0)

Reply:
Sorry, that's leftovers from a test I did. Fixed.